### DISCUSSION TITLE
Large HTML output on SSR

### DISCUSSION THREAD

**Discussion by SasanFarrokh** [tailwindlabs/tailwindcss#1602] [General]
If we render our app on server, initial html page will have a very large output since we have lots of classes, iterated blocks, and...
Is it worth to use tailwindcss on SSR applications?

**Comment by TrentFoulds** [comment]
The process where a user asks the server for some url, for example: https://www.mywebsite.com/how-to-ssr and the server responds with an html file, like the one shown in the image above. As simple as that, so remember, whenever some of your colleagues or some article is talking about SSR, don’t be afraid: It’s basically talking about a server that sent you file with html text in it (and maybe some headers attached). As a result, the browser has an html file that can process.
[Run Your Pool App](https://www.runyourpool.net/)
